lib/fix-me.js

```diff
@@ -18,10 +18,10 @@ FixMe.prototype.runEngine = function(){
       self = this;
 
   if (fs.existsSync('/config.json')) {
-    var overrides = JSON.parse(fs.readFileSync('/config.json'));
-
-    for (var prop in overrides) {
-        config[prop] = overrides[prop];
+    var user_config = JSON.parse(fs.readFileSync('/config.json'));
+    config.include_paths = user_config['include_paths'];
+    if (user_config.config && user_config['config']['strings']) {
+      config.strings = user_config['config']['strings']
```

Comment:
I think we can keep the `for (var ..` form in case we want to add new config keys later.

WDYT about this:

```js
config.include_paths = user_config.include_paths

if (user_config.config) {
  for (var prop in user_config.config) {
    config[prop] = user_config.config[prop]
  }
}
```